### PR TITLE
Create top_k_frequent_elements.cpp

### PR DESCRIPTION
**Pull request by shivangi-04**
Top K Frequent Elements in C++
#164

### CODE REVIEW

File: Code/C++/top_k_frequent_elements.cpp [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    * The unordered map is used for storing the frequency of each element
+    * The priority queue is used to store the element along with their frequencies in pairs
+    * The priority queue is implemented as a Min Heap to get the Top K Frequent elements
+*/
+
+#include <iostream>
+#include <vector>
+#include <queue>
+#include <unordered_map>
+
```
Comment (rudrakshi99) [inline Code/C++/top_k_frequent_elements.cpp:13]: Use `using namespace std;` instead of using `std` at every place.

**Review by rudrakshi99 — CHANGES_REQUESTED** [review]

**Comment by shivangi-04** [comment]
Please review the changes @rudrakshi99

**Review by rudrakshi99 — APPROVED** [review]

**Comment by rudrakshi99** [comment]
Update the list in the heap folder.

**Review by tarun26091999 — CHANGES_REQUESTED** [review]
link it to appropriate folder.

**Comment by shivangi-04** [comment]
Updated the list. Please check @rudrakshi99 @tarun26091999

**Comment by shivangi-04** [comment]
@rudrakshi99 @tarun26091999 Please check

**Pull request merged**